### ISSUE TITLE
[A11y]: Audit user journeys Master List

### ISSUE THREAD

**Issue by st0nebraker** [sourcegraph/about#5464]
As part of our goal to obtain WCAG 2.1 AA Accessibility compliance by the end of FY23, we will need to audit all major user journeys within the Marketing site, log accessibility issues, and plan to fix as many as we can.

This issue reflects the **auditing** stage of this work.

See [#31475](https://github.com/sourcegraph/sourcegraph/issues/31475) for more.

### How to use this issue

For planning purposes, we will start by auditing each new standardized layout by page type. Each team member should plan to set aside at least half a day per sprint to dedicate to conducting a new audit.

During an audit, a team member can try to fix any trivial issues encountered as they are conducting the audit. For larger, non-trivial issues or issues with design questions, a team member should record the issue using the [accessibility issue template](https://github.com/sourcegraph/about/issues/new?assignees=&labels=accessibility%2Cwcag%2F2.1%2Cwcag%2F2.1%2Ffixing&template=accessibility_issue.yaml&title=%5BAccessibility%5D%3A+) to follow up on in the **fixing stage**. For issues with straightforward requirements and steps to reproduce, consider setting the ticket aside to delegate to GitStart.

An issue should be marked as done when each type of audit is complete and all accessibility issues identified in it have been logged.

- [x] ~~#5500~~
- [x] ~~https://github.com/sourcegraph/about/issues/5465~~
- [x] ~~https://github.com/sourcegraph/about/issues/5466~~
- [x] ~~https://github.com/sourcegraph/about/issues/5467~~
- [x] ~~#5503~~
- [x] ~~#5519~~
- [x] ~~#5518~~
- [ ] #5517 🚧 
- [ ] #5501 🚧 
- [ ] #5515
- [ ] #5502 
- [ ] #5516
